umh.docs.umh.app/content/en/docs/production-guide/installation/flatcar-installation-virtual-machine.md

```diff
@@ -0,0 +1,159 @@
+---
+title: "Flatcar Installation (Virtual Machine)"
+content_type: task
+description: |
+    This page describes how to deploy the United Manufacturing Hub on Flatcar
+    Linux in a virtual machine.
+weight: 50
+aliases:
+   - /docs/production-guide/installation/installation-guide-flatcar/
```

Comment:
do two aliases work? same alias as flatcar-installation-bare-metal? which one is then used?

Reply:
In my tests it is kind of random. I thought that it'd better to get redirected to either one of the guides, but if you think that that slug should redirect to a specific one then I can remove it